torch/utils/inference/run.py

```diff
@@ -0,0 +1,569 @@
+r"""
+This is a script for launching PyTorch inference on Intel(R) Xeon(R) Scalable Processors with optimal configurations.
+Single instance inference, multi-instance inference are enabled.
```

Comment:
Could you add a description of what an "instance" is in this context (or switch to using "process"). For folks used to cloud, "instance" refers to a type of machine or node. So multi-instance implies multi-machine inference.

Reply:
This script will be executed as a process, and will invoke multiple (`--ninstances` or `number of cores/--ncore_per_instance`) "instances". each "instance" will also be executed with multiple threads. "instance" here is kind of a group of multiple threads, not a real process. Probably adding description is a good idea.

Reply:
To align with `--ncore_per_instance`, I think it might be better to use `--ninstances`. What do you think?